src/utilities/httpclient.cc

```diff
@@ -94,18 +93,17 @@ void HttpClient::setCerts(const std::string& ca, CryptoSource ca_source, const s
 
   if (ca_source == kPkcs11) {
     throw std::runtime_error("Accessing CA certificate on PKCS11 devices isn't currently supported");
-  } else {  // ca_source =kFile
-    std::unique_ptr<TemporaryFile> tmp_ca_file = std::make_unique<TemporaryFile>("tls-ca");
-    tmp_ca_file->PutContents(ca);
-    curl_easy_setopt(curl, CURLOPT_CAINFO, tmp_ca_file->Path().c_str());
-    tls_ca_file = std::move_if_noexcept(tmp_ca_file);
-  }
+  }  // ca_source =kFile
```

Comment:
This comment either belongs on a separate line or can be removed entirely.

Reply:
ok